src/components/buttons/LikeVideoBtn.js

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios';
+import { useData } from '../../contexts/DataContext';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+
+export function LikeVideoBtn({ video }) {
+    const [showDescription, setShowDescription] = useState(false);
+
+    const {
+        state: { likedVideos },
+        dispatch,
+    } = useData();
+
+    async function likeVideo() {
+        try {
+            const videoToPost = { ...video };
+            delete videoToPost.id;
```

Comment:
I have a suggestion here, I am not able to understand the need of line 17, you may include a comment here.